common/src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -74,9 +74,12 @@ public class BehaviorGroup extends CreationUpdateTimestamped {
     private boolean filterOutActions;
 
     @OneToMany(mappedBy = "behaviorGroup", cascade = CascadeType.REMOVE)
-    @JsonIgnore
```

Comment:
This is the one that allows to `return the event types associated to a behavior group`.
The rest mimics filtering out in the same places we filter out actions.

Reply:
I didn't see that one, sorry! The PR makes sense now 😄